recipes/cuda-version/meta.yaml

```
@@ -0,0 +1,29 @@
+{% set name = "cuda-version" %}
+{% set version = "12.0.0" %}
+{% set major_version = version.split('.')[0]|int %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: true  # [osx]
```

Since this is package is architecture independent (nothing different between Windows and Linux), we can just make it `noarch`

```suggestion
  noarch: generic
```